build.py

```diff
@@ -247,6 +256,8 @@ def do_operation(self, operation) :
 				exit_code = self.do_codecov()
 			else :
 				command = self.xcodebuild_command(operation, use_xcpretty)
+				if (os.environ.get('TRAVIS') == "true" and operation == "build" and self.use_build_wrapper == "true") :
+					command = "build-wrapper-macosx-x86 --out-dir build-wrapper-output " + command
 				print command
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Replace print statement by built-in function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=python%3APrintStatementUsage)